hack/release-scripts/image-digests-template.yaml

```diff
@@ -0,0 +1,43 @@
+# This file acts as the source of truth for the driver and sidecar image tags and digests used by the rest of the repository.
```

Comment:
Correct this line to not refer to a source of truth. (After convo with Eddie)

Reply:
This is marked as resolved, but the file hasn't been edited